1.8/administration/installing/cloud/aws/upgrading.md

```diff
@@ -0,0 +1,73 @@
+---
+post_title: Upgrading
+menu_order: 11
+---
+
+## Summary
+
+This document provides instructions for upgrading a DC/OS cluster from version 1.7 to 1.8 using AWS cloudformation templates. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation on AWS] before proceeding.
+
+**Important:**
+
+- The upgrade process described involves deleting all the instances, it will NOT save any persistent data. It is recommended creating a second cluster and migrate the services to save persistent data.
```

Comment:
Replace: It is recommended creating a second cluster and migrate the services to save persistent data.

With: If you have services which persist data locally to the cluster and the data should be preserved, it is recommended to create a second cluster running the new version of DC/OS and migrate the services and data to the new cluster.